en-US/Characters_Codex/06_Spellcasting/Spell_Descriptions/P/Prismatic_Spray.md

```diff
@@ -24,12 +43,12 @@ Roll a d8 for each target to determine the ray that affects it.
   The target takes `10d6` cold damage.
 6. **Indigo:**
   The target is [restrained](#Conditions_restrained) and at the end of each of its turns it makes a Constitution saving throw.
-  Once it accumulates two failed saves it permanently turns to stone, or when it accumulates two successful saves the effect ends.
+  Once it accumulates three failed saves it permanently turns to stone, or when it accumulates three successful saves the effect ends.
```

Comment:
This is like with _Stone to Flesh_; A5e changed it to two saves. We changed it back there (and A5e _didn't_ change it for _Prismatic Wall_), so I changed it back here as well.